src/main/java/com/joona/config/LibraryApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.joona.config;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan("com.joona")
```

Comment:
Please refer the documentation of "@SpringBootApplication". It houses Component scan within. You can specify the base package details within @SpringBootApplication(scanBasePackages = "com.joona)